<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainPage</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>SerialPort</string>
   </property>
   <widget class="QComboBox" name="cbSerialPortList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSerialOpen">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Network</string>
   </property>
   <widget class="QLineEdit" name="leNetPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10001</string>
    </property>
    <property name="placeholderText">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leNetIP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
    <property name="placeholderText">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNetConnect">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
